lib/mule.js

```diff
@@ -113,11 +113,26 @@ function mulelink(guid) {
 	return l;
 }
 
-var VAULTORDER = [34, 32, 30, 28, 31, 33, 35,
-                  27, 25, 21, 20, 22, 26, 29,
-                  23, 17, 13, 11, 14, 18, 24,
-                  16,  9,  6,  4,  7, 10, 19,
-                  12,  5,  2,  1,  3,  8, 15];
+var VAULTORDER = [
```

Comment:
No, I checked (at least for the first 44 chests) this is how it looks ingame. there are now 3 blocks: 2 5x6 and one 5x4 in the middle.